08LessonEight/homework/exercise-06.html

```diff
@@ -1,10 +1,65 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <title>Exercise 06</title>
-</head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="css/exercise-06-style.css">
+
+<head>
+    <title>Exercise 08</title>
+</head>
+
+<body>
+
+    <main>
+
+        <div>
+            <img class="responsive" src="img/truck.jpg" alt="Yikes" />
+            <h2>SAFE & SECURE DELIVERY</h2>
+            <p>
+                Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+
+            </p>
+            <span>Read more</span>
+
+        </div>
+        <div>
+            <img class="responsive" src="img/plane.jpg" alt="Yikes" />
+            <h2>SAFE & SECURE DELIVERY</h2>
+            <p>
+                Integer congue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
+
+            </p>
+            <span>Read more</span>
+        </div>
```

Comment:
Great job! Very clean html, use more semantic tags instead of just generic DIVs. For example, you could have used `<section>` `<aside>` etc.